src/js/entities-service/entities/forms.js

```diff
@@ -62,14 +61,20 @@ const _Model = BaseModel.extend({
     return size(submitReducers) ? submitReducers : [defaultSubmitReducer];
   },
   getWidgets() {
-    const formWidgets = get(this.get('options'), 'widgets');
+    const formWidgets = get(this.get('options'), ['widgets', 'widgets']);
 
-    return Radio.request('entities', 'widgets:collection', collectionOf(get(formWidgets, 'widgets'), 'id'));
+    if (!formWidgets) return Radio.request('entities', 'widgets:collection');
+
+    const allWidgets = Radio.request('bootstrap', 'widgets');
+
+    const widgets = map(formWidgets, slug => {
+      return allWidgets.find({ slug }).omit('id');
```

Comment:
all defined widgets should always be available.  `omit`ing `id` here creates a json object so that the widget collection can calculate the `modelId` for purposes of display.  It's possible in the future we may want the request to be `widgets:collection:forDisplay` or something where the collection is different than the standard collection.